examples/buildkit0/buildkit.go

```diff
@@ -18,7 +18,7 @@ func main() {
 	var opt buildOpt
 	flag.StringVar(&opt.target, "target", "containerd", "target (standalone, containerd)")
 	flag.StringVar(&opt.containerd, "containerd", "master", "containerd version")
```

Comment:
Can we fix containerd revision as well? (for more deterministic benchmark)

Reply:
Done